test/src/test/java/org/axonframework/test/aggregate/FixtureTest_StateStorage.java

```diff
@@ -61,6 +63,27 @@ void testCreateStateStoredAggregate() {
                .expectState(aggregate -> assertEquals("message2", aggregate.getMessage()));
     }
 
+    @Test
+    void testGivenCommandsForStateStoredAggregate() {
+        fixture
```

Comment:
Identation is a bit odd on this tests, not sure why.
I would expect it to be:
```
fixture.useStateStorage()
       .givenCommands ...
```

Reply:
Sorry for that. I just fixed the indentation in this file.